integrations/mailer/mailer.py

```diff
@@ -468,6 +469,12 @@ def main():
     OPTIONS['endpoint'] = os.environ.get('ALERTA_ENDPOINT') or OPTIONS['endpoint']  # nopep8
     OPTIONS['key'] = os.environ.get('ALERTA_API_KEY') or OPTIONS['key']
     OPTIONS['smtp_password'] = os.environ.get('SMTP_PASSWORD') or OPTIONS['smtp_password']  # nopep8
+
+    if os.environ.get('SMTP_USERNAME'):
+        OPTIONS['smtp_username'] = os.environ.get('SMTP_USERNAME')
+    elif not OPTIONS['smtp_username']:
+        OPTIONS['smtp_username'] = OPTIONS['mail_from']
```

Comment:
Wouldn't this work just as well ...
```
OPTIONS['smtp_username'] = os.environ.get('SMTP_USERNAME') or OPTIONS['smtp_username'] or OPTIONS['mail_from']
```
and is more consistent with how the other settings are determined.

Also for consistency, it should probably go on line 470 between `OPTIONS['key']` and `OPTIONS['smtp_password']`.